src/rgw/rgw_lc.h

```diff
@@ -164,21 +206,26 @@ class LCRule
     dm_expiration = _dm_expiration;
   }
 
+  void set_filter(LCFilter* _filter){
```

Comment:
good catch, ideally I guess this would be better as a boost::optional maybe, but will check for nullptr now